scripts/gas-refund-program/transactions-indexing/txs-covalent.ts

```diff
@@ -0,0 +1,65 @@
+import { covalentClient } from '../data-providers-clients';
+import { CovalentAPI, CovalentTransaction } from '../types';
+import { queryPaginatedData, QueryPaginatedDataParams } from '../utils';
+
+
+interface GetContractTXsByNetworkInput {
+  chainId: number;
+  contract: string;
+  startTimestamp: number;
+  endTimestamp: number;
+}
+
+export const covalentGetTXsForContract = async ({
+  startTimestamp,
+  endTimestamp,
+  chainId,
+  contract
+}: GetContractTXsByNetworkInput): Promise<CovalentTransaction[]> => {
+
+  const covalentAddressToTransaction = (txCov: CovalentAPI.Transaction): CovalentTransaction => ({
+    txHash: txCov.tx_hash,
+    txOrigin: txCov.from_address,
+    txGasPrice: txCov.gas_price.toString(),
+    txGasUsed: txCov.gas_spent.toString(),
+    blockNumber: txCov.block_height,
+    // convert time to unixtime (seconds)
+    timestamp: (new Date(txCov.block_signed_at).getTime() / 1000).toString(),
```

Comment:
why not `(+txCov.block_signed_at/1000).toString()` ?

Reply:
covalent doesn't return unix time, but a literal utc style timestamp eg 2022-04-17T18:59:13Z